controllers/user-controller.js

```diff
@@ -0,0 +1,11 @@
+const userServices = require('../services/user-service');
+
+const userController = {
+  signUp: (req, res, next) => {
+    userServices.signUp(req, (err, data) =>
```

Comment:
adapt async function pattern instead of callback pattern
(actually, your service has been an async function)

Reply:
```js
  signUp: async (req, res, next) => {
    const input = getFromReq(req);

    const user = await userServices.signUp(input);
    const records = await recordService.getByUserId(user.id)
    err ? next(err) : res.status(200).json({ status: 'success', data: { user, records } })
  },

```

Reply:
ok